### ISSUE TITLE
Control dumb self regulating Chlorinator with a relay.

### ISSUE THREAD

**Issue by cb88** [tagyoureit/nodejs-poolController#839]
I was considering setting up a system to control my above ground 14k gal pool with a ControlOMatic MegaChlor-CD which is a relatively simple device you just drop in your pool (provide it with 12V set the chlorination rate via buttons) 

Would it be possible to add automatic support for turning on/off a relay to control this device during hours where flow is occurring?  It might also make sense to notify if chlorination rate is too low/high etc... for manual adjustment for such devices.

Basically it would just turn the device on whenever flow is detected, and leave it on unless chlorine levels reach a set point the device should be self regulating though so in theory should never hit this.

**Comment by tagyoureit** [comment]
Are you not able to keep a consistent chlorination level (and I use consistent lightly because ORP changes with temp, # of swimmers, flow, etc)?  You could certainly wire something up in combination with [relayEquipmentManager](https://github.com/rstrouse/relayEquipmentManager/) but by the time you are invested in a RPI, outdoor enclosure, ORP sensor, relays, etc the cost might not be worth the effort just to turn on/off this one device.

**Comment by cb88** [comment]
True. I'm still in the design phase of my setup but am just looking for some more smarts and monitoring that don't have subscription cost, and are well made, I have a good idea of the costs involved, and its a bit experimental anyway so bound to cost me some extra here or there.

Current design is a bestway 15x32 above ground + DE filter W3EC50AC + Vevor solar pump + stock cheapo pump (these would require some valves to switch between them I won't bother if the solar pump has enough turnover but I might want to run it at night etc... ). I expect the solar pump to handle most of the turnover those its the 1.5Hp model. I plan on modifying the bestway hardware a bit to reduce corrosion even though they claim saltwater is fine.

I may just run the chlorinator off a timer as the cheapest option, this would allow it to get power from cheap charge controller in parallel with the pumps panels.
